### PR TITLE
rpmrebuild easyconfig

### PR DESCRIPTION
**Pull request by damianam**
Self descriptive title

### CODE REVIEW

**Comment by damianam** [comment]
ping

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/r/rpmrebuild/rpmrebuild-2.11.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+version = "2.11"
+
+homepage = 'http://rpmrebuild.sourceforge.net/'
+description = """rpmrebuild is a tool to build an RPM file from a package that has already been 
+   installed in a basic use"""
+
+toolchain = {'version': 'dummy', 'name': 'dummy'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%s-%s.tar.gz' % (name.lower(), version)]
```
Comment (boegel) [inline easybuild/easyconfigs/r/rpmrebuild/rpmrebuild-2.11.eb]: please use `SOURCELOWER_TAR_GZ` instead:

```python
sources = [SOURCELOWER_TAR_GZ]
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7a100712f8fb90098327 for a full test report.

**Comment by boegel** [comment]
@damianam: small style remark, otherwise good to go

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2f122bc690d60d8411ea for a full test report.

**Comment by boegel** [comment]
Going in, thanks @damianam!

**Comment by damianam** [comment]
My pleasure!

**Pull request merged**